opendp 0.15.0 failed on docs.rs

https://docs.rs/crate/opendp/0.15.0/builds: Docs build failed

```
[INFO] [stderr] 79 |         use core::unicode::{Case_Ignorable, Cased};
[INFO] [stderr]    |                                             ^^^^^ no `Cased` in `unicode`
````

on https://dev-doc.rust-lang.org/beta/core/unicode/fn.Cased.html it says:

> This is a nightly-only experimental API. (unicode_internals)

https://docs.rs/crate/opendp/0.15.0/builds/3327265

<details>

```

# error kind
ExecutionFailed


# pre-build errors
command failed: exit status: 101



# rustc version
rustc 1.97.0-nightly (9eb3be26b 2026-05-18)


# docs.rs version
docsrs 0.0.0 (e50152ed411bb913753b1dfd203f22cb8711f097 2026-05-17 )


# build log
[INFO] running `Command { std: "docker" "exec" "-e" "SOURCE_DIR=/opt/rustwide/workdir" "-e" "CARGO_HOME=/opt/rustwide/cargo-home" "-e" "RUSTUP_HOME=/opt/rustwide/rustup-home" "-e" "CARGO_TARGET_DIR=/opt/rustwide/target" "-e" "DOCS_RS=1" "-w" "/opt/rustwide/workdir" "--user" "1001:1001" "67146d69ea4b2673b53169f6255aca4b5b5c4e5e1ff572046cda9499df9e157c" "/opt/rustwide/cargo-home/bin/cargo" "+nightly" "rustdoc" "--lib" "-Zrustdoc-map" "--features" "use-openssl derive untrusted polars" "--no-default-features" "--config" "build.rustdocflags=[\"--cfg\", \"docsrs\", \"--html-in-header\", \"katex.html\", \"--document-private-items\", \"-Z\", \"unstable-options\", \"--emit=invocation-specific\", \"--resource-suffix\", \"-20260518-1.97.0-nightly-9eb3be26b\", \"--static-root-path\", \"/-/rustdoc.static/\", \"--cap-lints\", \"warn\", \"--extern-html-root-takes-precedence\"]" "--offline" "-Zunstable-options" "--config=doc.extern-map.registries.crates-io=\"https://docs.rs/{pkg_name}/{version}/x86_64-unknown-linux-gnu\"" "-Zrustdoc-scrape-examples" "-j6" "--target" "x86_64-unknown-linux-gnu", kill_on_drop: false }`
[INFO] [stderr] warning: target filter specified, but no targets matched; this is a no-op
[INFO] [stderr]     Checking polars-ops v0.52.0
[INFO] [stderr] error[E0432]: unresolved import `core::unicode::Cased`
[INFO] [stderr]   --> /opt/rustwide/cargo-home/registry/src/index.crates.io-1949cf8c6b5b557f/polars-ops-0.52.0/src/chunked_array/strings/case.rs:79:45
[INFO] [stderr]    |
[INFO] [stderr] 79 |         use core::unicode::{Case_Ignorable, Cased};
[INFO] [stderr]    |                                             ^^^^^ no `Cased` in `unicode`
[INFO] [stderr] 
[INFO] [stderr] error[E0603]: function `Case_Ignorable` is private
[INFO] [stderr]   --> /opt/rustwide/cargo-home/registry/src/index.crates.io-1949cf8c6b5b557f/polars-ops-0.52.0/src/chunked_array/strings/case.rs:79:29
[INFO] [stderr]    |
[INFO] [stderr] 79 |         use core::unicode::{Case_Ignorable, Cased};
[INFO] [stderr]    |                             ^^^^^^^^^^^^^^ private function
[INFO] [stderr]    |
[INFO] [stderr] note: the function `Case_Ignorable` is defined here
[INFO] [stderr]   --> /opt/rustwide/rustup-home/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/unicode/mod.rs:11:16
[INFO] [stderr]    |
[INFO] [stderr] 11 | pub(crate) use unicode_data::case_ignorable::lookup as Case_Ignorable;
[INFO] [stderr]    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[INFO] [stderr] 
[INFO] [stderr] Some errors have detailed explanations: E0432, E0603.
[INFO] [stderr] For more information about an error, try `rustc --explain E0432`.
[INFO] [stderr] error: could not compile `polars-ops` (lib) due to 2 previous errors
[INFO] running `Command { std: "docker" "inspect" "67146d69ea4b2673b53169f6255aca4b5b5c4e5e1ff572046cda9499df9e157c", kill_on_drop: false }`
```

</details>

--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -51,6 +51,8 @@ bitvec = "1.0"
 #
 # The features track the features enabled in the Rust build for the Polars Python crate.
 # Having matching features is necessary for cross-binary DSL compatibility.
+#
+# TODO: re-enable docs.rs polars once https://github.com/pola-rs/polars/issues/27105 is released
 polars = { version = "=0.52.0", features = ["full", "meta", "cutqcut", "row_hash"], optional = true }
 polars-python = { version = "=0.52.0", optional = true }
 
@@ -145,7 +147,7 @@ crate-type = ["rlib", "cdylib", "staticlib"]
 
 [package.metadata.docs.rs]
 no-default-features = true
-features = ["use-openssl", "derive", "untrusted", "polars"]
+features = ["use-openssl", "derive", "untrusted"]
 rustdoc-args = [
     # so that latex renders
     "--html-in-header", "katex.html",

